src/components/ComponentDetails/PropTable/index.tsx

```diff
@@ -38,7 +39,11 @@ const PropTable: React.FC<PropTableProps> = ({ propData }) => {
       propData.map((prop) => ({
```

Comment:
Let's destructure props so passing the props is a bit cleaner
`propData.map({ docs, type, required }) => ({`
`<PropDescription description={docs} type={type} required={required} />`